src/app/utils/persistentStorageTemplate.ts

```diff
@@ -0,0 +1,54 @@
+export const fileStore = `"file-store": {
+  "shared": false,
+  "data": {
+    "path": "data"
+  },
+  "index": {
+    "path": "index"
+  },
+  "write-behind": {
+    "modification-queue-size": "2048"
+  }
+}`;
+
+export const remoteStore = `"remote-store": {
+  "cache": "mycache",
+  "raw-values": "true",
+  "remote-server": [
+    {
+      "host": "one",
+      "port": "12111"
+    },
+    {
+      "host": "two"
+    }
+  ],
+  "connection-pool": {
+    "max-active": "10",
+    "exhausted-action": "CREATE_NEW"
+  }
+}`;
+
+export const sqlStore = `"table-jdbc-store": {
+  "dialect": "H2",
+  "shared": "true",
+  "table-name": "books",
+  "schema": {
+    "message-name": "books_value",
+    "package": "library"
+  }
+}`;
+
+export const jdbcStore = ``;
```

Comment:
Can't find example for JDBC Storage type, and please if you could verify all other templates

Reply:
```json
"string-keyed-jdbc-store": {
		"dialect": "H2",
		"string-keyed-table": {
			"prefix": "ISPN_STRING_TABLE",
			"create-on-start": true,
			"id-column": {
				"name": "ID_COLUMN",
				"type": "VARCHAR(255)"
			},
			"data-column": {
				"name": "DATA_COLUMN",
				"type": "BINARY"
			},
			"timestamp-column": {
				"name": "TIMESTAMP_COLUMN",
				"type": "BIGINT"
			},
			"segment-column": {
				"name": "SEGMENT_COLUMN",
				"type": "INT"
			}
		},
		"connection-pool": {
			"connection-url": "jdbc:h2:mem:infinispan",
			"driver": "org.h2.Driver",
			"username": "sa",
			"password": "changeme"
		}
	}
```